source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
+</div>
 
-Here's the summary of what you *must* do to secure your Home Assistant system:
+### Initial Setup
+
+After installation we strongly recommend you perform the following tasks before using the system.  
+
+| Task | Justification |
+| --- | --- |
+| Setup [encrypted remote access](/docs/configuration/securing/#remote-access). | Setup HTTPS to encrypt your communication.  HTTP will allow eavesdropping on your passwords. |
+| Secure [access methods](docs/configuration/securing/#extras-for-manual-installations) | Certain installation methods have dependencies on SSH and other access methods. It is best to remove or secure these. |
+| Ensure all [Updates](https://www.home-assistant.io/common-tasks/os/#update) are applied. | Running older versions of Home Assistant can provide vulnerabilities which may be exploitable. |
+
+
+### Configuration Changes
+Configuration changes carry risk.  These risks can be reduced by performing the following tasks. 
+
+| Task | Justification|
+| --- | --- |
+| Perform pre- and post-[Snapshots](https://www.home-assistant.io/common-tasks/os/#snapshots). | In the event a configuration change needs to be reverted, you can easily revert using snapshots. |
+| Use built-in [secrets](/docs/configuration/secrets/) handling. | When requesting help from the [forums](https://community.home-assistant.io/) you will want to share your configuration. Secrets reduces your risk of exposure of sensitive data. |
+| Run [configuration checks](https://www.home-assistant.io/common-tasks/os/#configuration-check). | After making configuration changes, you should ensure your configuration is valid so it will operate as expected. | 
+
+### Maintenance
+Your system is your own responsibility.  Maintenance should be performed regularly to ensure your system is secure. Many of these tasks can be automated. Your specific setup will dictate additional tasks. 
+
+| Task | Justification|
+| --- | --- |
+| Check [security advisories](www.home-assistant.io/blog/categories/public-service-announcement/) | An informed user is able to make better decisions than an uninformed user. |
+| Read [Release Notes](https://github.com/home-assistant/operating-system/releases/) before updating. | Breaking changes can occur and you can read the release notes to identify problems and find workarounds. |
+| Perform [Snapshots](https://www.home-assistant.io/common-tasks/os/#snapshots). | Snapshots are a quick way to get your system back up and running in the event of failure. |
+| Ensure all [Updates](https://www.home-assistant.io/common-tasks/os/#update) are applied.  | Running older versions of Home Assistant can provide vulnerabilities which may be exploitable. |
+| Update  [Encryption Certificates](/docs/configuration/securing/#remote-access). | TLS certs expire on a regular basis. Once expired your browser will not recognize them as valid. |
+| Prune Integrations/add-ons | Remove unused Integrations and add-ons to reduce your risk of exposure. |
```

Comment:
https://alerts.home-assistant.io/
This is a good link to have, which should probably be at the top of this list.

Reply:
Great suggestion!  Thanks!